var/spack/repos/builtin/packages/gpi-2/package.py

```diff
@@ -67,29 +68,29 @@ class Gpi2(AutotoolsPackage):
     depends_on('slurm', when='schedulers=slurm')
 
     conflicts('%gcc@10:', when='@:1.3.2', msg='gcc>10 is not supported')
-    conflicts('schedulers=slurm', when='@:1.3.2', msg='Slurm is not supported')
+    conflicts('schedulers=slurm', when='@:1.3.3', msg='Slurm is not supported')
 
     def set_specific_cflags(self, spec):
         if spec.satisfies('@:1.4.0%gcc@10.1.0:'):
```

Comment:
Hi @mzeyen1985, the changes in version 1.3.3 allow to set:
```suggestion
        if spec.satisfies('@1.4.0%gcc@10.1.0:'):
```
otherwise, `-fcommon` will still be used.

Reply:
Applied the suggested changes.